public/build/vendors~login~program_comments~program_description~register~report~request~reset.js

```diff
@@ -0,0 +1,4195 @@
+(window["webpackJsonp"] = window["webpackJsonp"] || []).push([["vendors~login~program_comments~program_description~register~report~request~reset"],{
```

Comment:
Those public/build/* files seem all autogenerated. Should they really be in the repository? or only generated locally?

Reply:
Discussed with @HCrane already, for some reason if those files are not pushed, the javascript does not work (see https://github.com/Catrobat/Catroweb/pull/1070 where I did not push them). Tests work if only entrypoints.json is pushed, but then we can't see the behavior on web-test